src/provider/ClickstreamProvider.ts

```diff
@@ -191,4 +194,13 @@ export class ClickstreamProvider implements AnalyticsProvider {
 	flushEvents(eventRecorder: EventRecorder) {
 		eventRecorder.flushEvents();
 	}
+
+	sendEventsInBackground(isWindowClosing: boolean) {
+		if (
+			!(BrowserInfo.isFirefox() && isWindowClosing) &&
```

Comment:
Firefox dose not support this feature?

Reply:
When window is closing, Firefox can not sending events by setting `keep-alive`, see the [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#browser_compatibility) or [caniuse](https://caniuse.com/mdn-api_request_keepalive)